queries/snakemake/highlights.scm

```diff
@@ -0,0 +1,63 @@
+; inherits: python
+
+;; Compound directives
+["rule" "checkpoint" "module"] @keyword
+
+;; Simple directives
+(directive name: _ @keyword)
+
+
+;; Rule imports
+(rule_import
+  "use" @include
+  "rule" @include
+  "from" @include
+  "as"? @include
+  "with"? @include
+)
+
+
+;; Rule inheritance
+(rule_inheritance
+  "use" @keyword
+  "rule" @keyword
+  "with"? @keyword
+)
+
+
+;; Wildcard names
+;; as @field for consistency with attributes in wildcard interpolations
+(wildcard (identifier) @field)
```

Comment:
```suggestion
```

Field doesn't make much sense

Reply:
My main concern here is consistency of highlighting between the definition and interpolation of wildcards. The parameters to directives are `keyword_argument`s, in which `identifier` is highlighted as `@parameter` --> `(wildcard (identifier) @parameter)`?
Related: https://github.com/nvim-treesitter/nvim-treesitter/pull/4302#discussion_r1103862281

So to summarize this `@field` and the other `@keyword` issue, I want to achieve 3 things here:

1. Capture the hierarchical structure between directives and their parameters (`keyword` > `parameter/field`). Making the directives `field` loses that.
2. Maintain consistency between definitions and accessions. Dropping `field` (/`parameter`?) from wildcard identifiers loses that.
3. Avoid touching base Python as much as possible. Switching from `keyword` to `field` for directives and dropping `field` from wildcard identifiers maintains hierarchy, but loses consistency, unless `(keyword_argument (identifier)` is demoted from `parameter`.

Reply:
Updated queries to cut down on keyword use: https://github.com/nvim-treesitter/nvim-treesitter/pull/4302/commits/3b4275af2f11875b75de6432195d150acdbe48e7

Please re-open if you disagree.